### PR TITLE
issue: 800923 Remove and ignore some autoconf files

### PR DESCRIPTION
**Pull request by igor-ivanov**
No need in config.guess and config.sub files.
Note: ARM 64 bit CPU architecture (aarch64) support
was introduced in autoconf 2.69.

Signed-off-by: Igor Ivanov <igor.ivanov.va@gmail.com>

See original PR https://github.com/Mellanox/libvma/pull/119

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-redmine-issue/66/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/183/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
bot:retest

File: .gitignore [hunk 1]
```diff
@@ -26,7 +26,17 @@ config.h.in
 config.h.in~
```
Comment (OphirMunk) [inline .gitignore]: Please update commit message. Please note autoconf 2.69 is mentioned as required - which is not the case

Comment (igor-ivanov) [inline .gitignore]: Updated

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-redmine-issue/67/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/184/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-redmine-issue/68/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/185/ for details (Mellanox internal link).

**Pull request merged**